devel/spicy/reference.rst

```diff
@@ -602,6 +602,91 @@ optional on the Zeek-side, as they may not have been set during
 parsing. Unit variables are non-optional by default, unless declared
 as ``&optional`` in Spicy.
 
+.. _spicy_unit_export:
+
+Controlling created Zeek types
+""""""""""""""""""""""""""""""
+
+If needed the automatic unit type exporting described above can be customized. The general syntax for this is
+
+.. code-block:: spicy-evt
+
+    export SPICY_ID [with { [SIPCY_FIELD_NAME [&log], ]... }];
+    export SPICY_ID [as ZEEK_ID (with { [SIPCY_FIELD_NAME [&log], ]...) }];
+
+.. rubric:: Controlling exported fields
+
+Assume we are given a Spicy unit type ``foo::X``
+
+.. code-block:: spicy
+
+    module foo;
+
+    public type X = unit {
+        x: uint8;
+        y: uint8;
+        z: uint8;
+    };
+
+To create a Zeek type with only the field ``x`` to following ``export`` statements are equivalent:
```

Comment:
```suggestion
To create a Zeek type without the field ``x`` the following ``export`` statements are equivalent:
```

Reply:
Higher level, I find it confusing to group all these 4 cases together because they all have different purposes, whereas the "are equivalent" seems to suggest they are all doing the same and it doesn't matter which one to use. Suggestion: walk through the 4 cases invididually: (1) skip a field, (2) include only selected fields, (3) rename the record.

Reply:
Good idea. I have completed the grammar and added separate explanations of the different uses cases. With that there is no need to show the "equivalent" forms below. I kept the verbose use cases below, but now mark them _Example_.